submissions/NadiaVorontsova/dom-api-task/script.js

```diff
@@ -0,0 +1,69 @@
+window.addEventListener("DOMContentLoaded", () => {
+
+    const seriesCOD = [{
```

Comment:
In the loading function set only the initial function calls and event listeners.

Reply:
I removed load function, because i wrote script before closing body tag.